tools/library/src/reference/initialize_reference_operations.cu

```diff
@@ -43,45 +43,47 @@ namespace cutlass {
 namespace library {
 
 // note: init methods for the same op-class may be split into multiple to parallelize compilation
-void initialize_gemm_reference_operations_int4(Manifest &manifest);
-void initialize_gemm_reference_operations_int8_interleaved_32(Manifest &manifest);
-void initialize_gemm_reference_operations_int8_interleaved_64(Manifest &manifest);
-void initialize_gemm_reference_operations_int8_canonical(Manifest &manifest);
-void initialize_gemm_reference_operations_e4m3a_e4m3out(Manifest &manifest);
-void initialize_gemm_reference_operations_e5m2a_e4m3out(Manifest &manifest);
-void initialize_gemm_reference_operations_e4m3a_e5m2out(Manifest &manifest);
-void initialize_gemm_reference_operations_e5m2a_e5m2out(Manifest &manifest);
-void initialize_gemm_reference_operations_fp8in_fp16out(Manifest &manifest);
-void initialize_gemm_reference_operations_fp8in_bf16out(Manifest &manifest);
-void initialize_gemm_reference_operations_fp8in_fp32out(Manifest &manifest);
-void initialize_gemm_reference_operations_fp32out(Manifest &manifest);
-void initialize_gemm_reference_operations_fp_other(Manifest &manifest);
+// void initialize_gemm_reference_operations_int4(Manifest &manifest);
+// void initialize_gemm_reference_operations_int8_interleaved_32(Manifest &manifest);
+// void initialize_gemm_reference_operations_int8_interleaved_64(Manifest &manifest);
+// void initialize_gemm_reference_operations_int8_canonical(Manifest &manifest);
+// void initialize_gemm_reference_operations_e4m3a_e4m3out(Manifest &manifest);
+// void initialize_gemm_reference_operations_e5m2a_e4m3out(Manifest &manifest);
+// void initialize_gemm_reference_operations_e4m3a_e5m2out(Manifest &manifest);
+// void initialize_gemm_reference_operations_e5m2a_e5m2out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_fp16out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_bf16out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_fp32out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp32out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp_other(Manifest &manifest);
```

Comment:
RESOLVE THIS: Uncomment it before the merge. These are commented out for speed of development and reducing compilation times.